nomos-da/network/core/src/protocols/sampling/mod.rs

```diff
@@ -80,19 +80,30 @@ mod test {
             p1_addresses.into_iter().collect(),
         );
         let mut p2 = sampling_swarm(k2.clone(), neighbours, p2_addresses.into_iter().collect());
-
         let request_sender_1 = p1.behaviour().sample_request_channel();
         let request_sender_2 = p2.behaviour().sample_request_channel();
         const MSG_COUNT: usize = 10;
+        let channel1 = Indicator::new(MSG_COUNT);
+        let channel2 = Indicator::new(MSG_COUNT);
```

Comment:
Please use better naming, we already have `Indicator`, why are we using `channelX`? 😅

Reply:
Yeah, this one really needs to  be fixed...I copy-pasted from my previous version..apology